### ISSUE TITLE
Kvantum support

### ISSUE THREAD

**Issue by carlos22** [themix-project/oomox-gtk-theme#55]
Kvantum is a Qt5 svg based theme engine with allows you to easily theme qt5 applications (including lxqt and kde) [1]. For materia there is a Kvantum theme available [2] unfortionatly it is not compatilble with change_color.sh yet. Currently I try to get that working. QGnomePlatfrom [3] is a Qt plattfrom integration plugin which has support for automatically configure Kvantum themes if they have the same name as the current gtk+3 theme. This plugin is installed in by default at least on fedora ubuntu had some progress on this but it did not made it into 18.04 [4]. I think this is the only option to have consistent look across gtk and qt apps.

[1] https://github.com/tsujan/Kvantum/tree/master/Kvantum
[2] https://github.com/PapirusDevelopmentTeam/materia-kde/tree/master/Kvantum/Materia
[3] https://github.com/FedoraQt/QGnomePlatform/blob/master/src/gnomehintssettings.cpp#L232
[4] https://discourse.ubuntu.com/t/qt-integration-for-18-04/4323

**Comment by actionless** [comment]
i think for arc theme it will be easier to implement than for oomox theme

both materia and arc have assets.svg file with all the theme assets (the same as Kvantum theme requires)

while oomox theme using svg assets only for checkboxes and radios (and rest of the widgets drawing using theme engine/css)

**Comment by carlos22** [comment]
Ok. Well the colors differ a bit between the Kvantum themes of materia and arc i found. So this needs to be adapted to work with `change_colors.sh` script. I think it is better to have a somewhat similar theme with the same colors as having fusion theme for qt5 apps. The old numix qt4 qtcurve config was simliar, not the complete style but at least the same colors.

**Comment by actionless** [comment]
sorry, i didn't understood your point regarding fusion

**Comment by carlos22** [comment]
Oh fusion is just the default qt theme and therefore used by default if the theme does not provide a qt or kvantum theme (in case of gnome with QGnomePlatform).

**Comment by actionless** [comment]
but oomox theme is already generating Fusion palette

**Comment by carlos22** [comment]
Yes but that is only useable with qt5c as far as i know. There is not support in QGnomePlatform.

**Comment by actionless** [comment]
yeah, it seems to be the only one which is really working outside of some DE

feel free to convert that config to be comptible with some other tools which you use and share it
